Contribute/how-to-write-links.md

```diff
@@ -79,41 +79,53 @@ You do not have to create anchors. They're automatically generated at publishing
 
 Because include files are located in another directory, you must use longer relative paths. To link to an article from an include file, use this format:
 
-    [link text](../articles/folder/article-name.md)
+   ```markdown
+   [link text](../articles/folder/article-name.md)
+   ```
 
 ## Links in selectors
 
-If you have selectors that are embedded in an include--as does the Azure documentation team--use the following link structure:
+A selector is a navigation component that appears in a docs article as a drop down list. When a reader selects a value in the drop down, the browser opens the selected article. Typically the selector list contains links to closely related articles, for example the same subject matter in multiple programming languages or a closely related series of articles. 
```

Comment:
@JasonWHowell I'm not familiar with selectors either (I didn't write this section). Should we just delete this section, perhaps?

Reply:
@gewarren 
In azure-docs, there are around 85 of these: `[!div class="op_single_selector"]` in include files that can be added to the top of articles to provide a drop down (in lieu of tabs maybe). 

For example, https://github.com/MicrosoftDocs/azure-docs/blob/master/includes/active-directory-b2c-devquickstarts-web-switcher.md

Reply:
@TerryGLee do you remember this section?